### DISCUSSION TITLE
Multi-source and REST API datasource in project.

### DISCUSSION THREAD

**Discussion by Vladimir-Va** [nocodb/nocodb#393] [Features]
Please think the ability to add tables from other projects to one project.
Add the ability to work with a data source not only databases but also third-party REST APIs (for example, obtaining exchange rates, integrating data from your service, etc.).
I come from [phprunner](https://xlinesoft.com/blog/2020/03/20/working-with-third-party-rest-api/) is very helpfull.

**Comment by o1lab** [comment]
As mentioned in previous conversation in discord and github.

Adding data sources is a significant effort - this request was made on the first day of launch. I will create a longer roadmap and include it.

**Comment by o1lab** [comment]
Thanks for the mention of phprunner - will look into it.
